docs/release-notes/NuGet-6.10.md

```diff
@@ -0,0 +1,177 @@
+---
+title: NuGet 6.10 Release Notes
+description: Release notes for NuGet 6.10 including new features, bug fixes, and DCRs.
+author: kartheekp-ms
+ms.date: 5/13/2024
+ms.topic: conceptual
+---
+
+# NuGet 6.10 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.10**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.10](https://visualstudio.microsoft.com/downloads/) | [8.0.3xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.10
+
+* [Feature]: add command dotnet config set - [#12476](https://github.com/NuGet/Home/issues/12476)
```

Comment:
I think we need to include this issue: https://github.com/NuGet/Home/issues/12469 instead. 
Since it talks about the whole dotnet nuget config command.

Reply:
Make sense Nikolche. Fixed in the latest commit.